webevd/WebEVD/WebEVDServer.cxx

```diff
@@ -367,6 +367,11 @@ JSONFormatter& operator<<(JSONFormatter& json, const recob::Vertex& vtx)
 // ----------------------------------------------------------------------------
 JSONFormatter& operator<<(JSONFormatter& json, const simb::MCTruth& mct)
 {
+  // Don't show MCTruth for cosmic rays, which can be extremely
+  // lengthy. Ideally we should exclude them from the list entirely, but this
+  // requires less change to the structure of the code.
+  if(mct.Origin() == simb::kCosmicRay) return json << "\"\"";
```

Comment:
does this mean that the truth trajectories are no longer shown or some accompanying text is now longer printed out?

Reply:
Just the text (which is part of the display). It's basically a screen-length series of "mu"s and energies.

The truth trajectories are also an issue, since there are so many of them, and because of t0 they don't even align with their tracks in the detector. I wanted to make them toggleable, but it requires walking over the parentage tree and I haven't got around to that.